docs/gfm-alerts-to-admonitions/gfm_alerts_to_admonitions/__init__.py

```diff
@@ -0,0 +1 @@
+from .main import gfm_alert_to_admonition
```

Comment:
We're getting a linting error: https://buildbot.nix-community.org/#/builders/448/builds/1104/steps/1/logs/stdio

Reply:
I have no idea what that means xD

Reply:
I'll look it up

Reply:
See https://docs.astral.sh/ruff/rules/unused-import/ for more details